examples/ReactExample/run-android.sh

```diff
@@ -6,7 +6,7 @@ set -e
 PATH="/opt/android-sdk-linux/platform-tools:$PATH"
 
 # Inform the prepublish script to build Android modules.
-REALM_BUILD_ANDROID=1 npm install realm
+REALM_BUILD_ANDROID_PACKAGE=1 npm install realm
```

Comment:
The comment for this line states: `# Inform the prepublish script to build Android modules.`.
But `scripts/prepublish.js` has been deleted & the `prepublish` npm script removed - so is this env-setting still needed?

Reply:
good catch. removed it